esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -794,8 +794,7 @@ define([
             registry.byId(this.id + "CopyDropDown").set("disabled", !hasSelection);
             registry.byId(this.id + "RenameDropDown").set("disabled", !hasSelection);
             registry.byId(this.id + "AddtoDropDown").set("disabled", !hasSelection);
-            registry.byId(this.id + "AddtoDropDown").set("disabled", !hasSelection);
-            registry.byId(this.id + "DesprayDropDown").set("disabled", !hasSelection);
+            registry.byId(this.id + "DesprayDropDown").set("disabled", !hasSelection || dojoConfig.isContainer);
```

Comment:
Why is this disabled for container?  I would have expected exporting/despraying to still be relevant.

Reply:
I can leave it enabled if the expectation is that we will have Azure landing zone support for v8?

Reply:
This should be working when Shamser's PR is merged.